x/mobileproxy/README.md

```diff
@@ -1,7 +1,19 @@
-# Mobileproxy: Local Proxy Library for Mobile Apps
+# MobileProxy: Local Proxy Library for Mobile Apps
 
 This package enables the use Go Mobile to generate a mobile library to run a local proxy and configure your app networking libraries.
 
+**Content app without MobileProxy:**
+![image](https://github.com/user-attachments/assets/d96b209c-2198-4382-8bba-12efd8f75dbb)
```

Comment:
Improvements for the future: we may check in the images to the repository, so we don't need to depend on one specific user's content.